main.rb

```diff
@@ -0,0 +1,43 @@
+require './app'
+
+class Main
+  app = App.new
+
+  puts 'welcome to School Library App!'
+  puts 'Please choose an option by entering a number:'
+
+  loop do
+    puts "\n"
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+    puts "\n"
+
+    option = gets.chomp
+
+    case option
+    when '1'
+      app.books_list
+    when '2'
+      app.people_list
+    when '3'
+      app.create_person
+    when '4'
+      app.create_book
+    when '5'
+      app.create_rental
+    when '6'
+      app.show_rental
+    when '7'
+      puts puts "Thank you for using this app!\n\n"
+      break
+    else
+      puts 'Incorrect choice, Please choose between 1 and 7'
+    end
+  end
+end
+Main.new
```

Comment:
- [ ] Thank you for your effort. You are close to achieving your goal. As I requested in previous review, you must call the main function. Here is sample code: 

```# main.rb
def main
  app = App.new()

  response = nil

  # [OPTIONAL] you can remind the student about the ruby lessons about basic sintax like loops: https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/lessons/basic_syntax.md  
  # *Present the user with a list of options to perform.
  puts "Welcome to School Library App!\n\n"
  while response != "7"
    puts "Please choose an option by enterin a number:"
    puts "1 - List all books"
    puts "2 - List all people"
    puts "3 - Create a person"
    puts "4 - Create a book"
    puts "5 - Create a rental"
    puts "6 - List all rentals for a given person id"
    puts "7 - Exit"
    # *Lets users choose an option.
    response = gets.chomp

    case response
    when "1"
      app.list_books()
    when "2"
      app.list_people()
    when "3"
      app.create_person()
    when "4"
      app.create_book()
    when "5"
      app.create_rental()
    when "6"
      app.list_rentals_for_person_id()
    when "7"
      # *Have a way to quit the app.
      puts "Thank you for using this app!"
    end

    puts "\n"
  end
end

# 'main' method should be invoked at the end of the main.rb file, hint in this lesson https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/console_apps.md#console-app-entry-point
main()
```